### DISCUSSION TITLE
Incoming/Outgoing Emails - Google Mail?

### DISCUSSION THREAD

**Discussion by jfi** [chatwoot/chatwoot#10401] [Self hosted]
Hi everyone,

I've just started configuring chatwoot as a self-hosted instance. It looks like it'll be great for us in terms of data protection, etc.

I note that sending emails seems to suggest Mandrill, Sendmail, etc.

My current configuration is as follows: 

1. We're using mydomain.org (example) as our primary domain. mydomain.org is using Google Mail.
2. People email team@mydomain.org. I want to set this as an incoming email that'll come into an inbox.
3. When we reply, I ideally want emails to come from team@mydomain.org.
4. Chatwoot is at omnichannel.mydomain.org.

My `.env` file is set as follows (just the relevant bits, I think):

```
FRONTEND_URL=https://omnichannel.mydomain.org
MAILER_SENDER_EMAIL=mydomain <team@mydomain.org>
SMTP_DOMAIN=mydomain.org
SMTP_ADDRESS=smtp.gmail.com
SMTP_PORT=587
SMTP_USERNAME=team@mydomain.org
SMTP_PASSWORD=[app specific password]
SMTP_AUTHENTICATION=login
SMTP_ENABLE_STARTTLS_AUTO=true
SMTP_OPENSSL_VERIFY_MODE=peer
MAILER_INBOUND_EMAIL_DOMAIN=omnichannel.mydomain.org
RAILS_INBOUND_EMAIL_SERVICE=relay
```

At the moment, if I create a new contact and message, it sends out fine - so the SMTP settings are working. I'm still tweaking to get incoming mail working properly - on an inbox, it suggests a forwarding address such as "812e4cc38d3432f081f64a854a505e58@omnichannel.mydomain.org", but emails sent to this address aren't received.

We're not expecting huge volumes through. So, a few questions:

1. Is there anything preventing us from using Google Mail both ways?
2. I ideally don't want to have to send email from omnichannel.mydomain.org and have to warm that domain up for mail as well as mydomain.org.

Does anyone have any suggestions of the best way to approach this? Not at all opposed to using another provider, e.g. Mailgun or Mandrill, if needs be.

**Comment by sojan-official** [comment]
@jfi  rather than configuring this as an installation wide config, I would recommend configuring an email channel with in an account in your Chatwoot installation using the gmail settings 

https://www.chatwoot.com/docs/self-hosted/configuration/features/email-channel/google-workspace-setup


---- 

setting up email forwaring is required only if you want to use Chatwidget extensively with conversation continuity. in such cases you will have to use a provider like sendgrid or mail gun. as there is not possibility of generating the dynamic email address with google mail
